modules/deploy/pages/deployment-option/cloud/tiers.adoc

```diff
@@ -0,0 +1,10 @@
+= Throughput Tiers
+:description: Select a throughput tier for your Redpanda Cloud cluster.
+:page-context-links: [{"name": "BYOC", "to": "deploy:deployment-option/cloud/tiers.adoc" },{"name": "Dedicated", "to": "deploy:deployment-option/cloud/dedicated/dedicated-tiers.adoc" } ]
```

Comment:
I don't know enough about context links so maybe this isn't possible: I wonder if we should also add the "byoc" indication the BYOC page/URL. I don't think it's a big deal but maybe there is some benefit

Reply:
good idea! I've renamed it to byoc-tiers.adoc, so it appears in the URL.